azure-pipelines/windows_test_jobs.yml

```diff
@@ -11,13 +11,15 @@ parameters:
   - select
   - iocp
 
-
 jobs:
 - ${{ each pyver in parameters.pythonVersions }}:
   - job: ${{ format('windows_{0}_{1}reactor', pyver.key, parameters.reactor) }}
     displayName: ${{ format('Windows Py{0} {1}reactor', pyver.value, parameters.reactor) }}
     pool:
       vmImage: ${{ parameters.imageName }}
+    variables:
+      PYTHONUTF8: 1
+      PYTHONIOENCODING: "utf-8:surrogateescape"
```

Comment:
I think that these variables need a comment.
Why do we need to set them and why not use the default values.

if these variables are required to run twisted under Windows, I think that this should be reflected in the public documentation.

Reply:
@adiroiban Where in the public documentation?

Reply:
I would say that it should be somewhere in the installation part 
 https://twistedmatrix.com/documents/current/installation/index.html

Maybe create a new page to document the Windows runtime environment  requirements... as well as the async io event loop change that is required on Windows.

Reply:
OK, I added a comment.